.coveragerc

```
@@ -0,0 +1,16 @@
+[report]
```

```suggestion
[run]
branch = True

[report]
```
I run IDLE tests with 'branch = True' and consider this the proper choice.  I don't know if others dispute this or not.  OP Ammar Askar, in https://github.com/python/cpython/pull/8150#issuecomment-403193951, removed this option because this "caused some reliably reproducible test errors".  He did not list them or get back to "make bpo bugs for them later after investigating".  This puzzles me since this option should only affect the coverage report, not the test execution itself.  

Ammar's later comment https://github.com/python/cpython/pull/8150#issuecomment-536350378 seems relevant but I don't understand what he wrote.  He appears to say that codecov would pay attention to this file placed where proposed, but I know nothing about codecov or whether the linked outputs were run with this file and if so with or without the branch option.

In any case, I believe I added DO NOT MERGE because this branch option issue should be resolved first.